src/server/main.c

```
@@ -950,47 +1016,47 @@ int main(int argc, char **argv)
-                            bc_queue_unreliable(j, hbuf_own, hlen_own);
-                        } else if (j != i && mixed_rmt_len > 0) {
-                            bc_queue_unreliable(j, mixed_rmt, mixed_rmt_len);
-                        } else if (j != i && hlen_rmt > 0) {
-                            /* Fallback to 0x20-only if mixed build fails. */
-                            bc_queue_unreliable(j, hbuf_rmt, hlen_rmt);
-                        }
-                    }
-                }
-            }
+                    u8 hbuf_rmt[128];
+                    u8 rmt_next;
```

Clang (14.0.6)

[warning] variable 'rmt_next' is not initialized

(cppcoreguidelines-init-variables)